libs/ml/examples/word2vec_distributed_learning/main.cpp

```diff
@@ -129,6 +130,7 @@ int main(int argc, char **argv)
     cp.data                        = {client_data[i]};
     auto client =
         std::make_shared<Word2VecClient<TensorType>>(std::to_string(i), cp, console_mutex_ptr);
+    client->SetMaxUpdates(iterations_count);
     // TODO(1597): Replace ID with something more sensible
```

Comment:
This comment isn't in the right place any more! I think we decided not to do anything about this issue anyway, so maybe we can remove it?